src/persist/benches/writer.rs

```diff
@@ -313,6 +314,7 @@ pub fn bench_writes_blob_cache(c: &mut Criterion) {
 
     let updates = generate_updates();
     let size = get_encoded_len(updates.clone());
+    let updates = vec![updates.iter().collect::<ColumnarRecords>()];
```

Comment:
let's make generate_updates return a ColumnarRecords instead

Reply:
I'd rather not do that because generate_updates is also used to generate a vec of updates for calls to `bench_writes` (which uses a `WriteReqBuilder`)

Keeping all of the data generating functions tied to the same underlying function helps me make sure that the benchmarks that write unsealed batches and the benchmarks that write data to indexed and then call drain pending are writing the same exact data.

Reply:
yeah I noticed, but it's just as easy for bench_writes to go `ColumnarRecords` -> `Vec<(...)>` as for this to go `Vec<(...)>` -> `ColumnarRecords`. and I'd rather the benchmarks that only need a ColumnarRecords don't have to go through the allocs of the Vec thing (even in that's only happening in setup code)

Reply:
took me a bit to find the `write!` macro to actually generate keys in place, but done!